docs/APIReference-Relay.md

```diff
@@ -150,23 +150,20 @@ As an example, we can log each mutation that is sent to the server as follows:
 ```
 var DefaultNetworkLayer = Relay.DefaultNetworkLayer;
 
-var MutationLoggingNetworkLayer = {
-  // Use the defaults for other options.
-  ...DefaultNetworkLayer,
-
+class MutationLoggingNetworkLayer extends DefaultNetworkLayer {
   sendMutation(mutation) {
     // Log the query text exactly as it will be sent to the server.
     console.log(mutation.getQueryString());
     // Send the mutation using the default network implementation, but
     // log the response or error.
-    return DefaultNetworkLayer.sendMutation(mutation).then(
+    return super.sendMutation(mutation).then(
```

Comment:
How about updating this with approach from https://github.com/facebook/relay/pull/684/files#r50766453 ?